README.md

```diff
@@ -308,6 +308,42 @@ There are two possible workarounds for this:
 
   </details>
 
+### Using MacOS after removing `nix` while `nix-darwin` was still installed, network requests fail
+
+If `nix` was previously uninstalled (using the [reference documentation](https://nixos.org/manual/nix/stable/installation/uninstall))
```

Comment:
IIRC, this happened with our installer up until we fixed it too, so maybe just say "if it was uninstalled without uninstalling nix-darwin first", so that it covers both users of older versions of our installer and users of the official installer?